### ISSUE TITLE
Workflow that worked with v0.3 but does not with v0.4

### ISSUE THREAD

**Issue by tueda** [1arp/create-a-file-action#5]
The following usage of the action does not work with 0.4.2 [(log)](https://github.com/tueda/test-create-a-file-action/actions/runs/8567670337) or 0.4.4 [(log)](https://github.com/tueda/test-create-a-file-action/actions/runs/8567672829):
```yaml
      - uses: 1arp/create-a-file-action@0.4.4
        with:
          file: foo.txt
          content: |
            Hello
            World
```
```
Error: ENOENT: no such file or directory, mkdir ''
```
Indeed, with 0.3 [(log)](https://github.com/tueda/test-create-a-file-action/actions/runs/8567663637), this successfully creates `foo.txt` in the root of the repository directory.

**Comment by 1arp** [comment]
Hi @tueda,
Thanks for bringing this forward.
You are right it seems that the issue is related to https://github.com/actions/toolkit/issues/361
I will patch this ASAP. ( ECD : end of this weekend )
Sorry for breaking your workflow.

The `getInput` function used to parse the new `isAbsoluteParam` parameter parses it as a string rather than a boolean, leading to reading the default value as a string which is `truthy`. The correct way to parse the input will be to use `getBooleanInput`.

**Comment by 1arp** [comment]
The issue should be fixed with latest `0.4.5` release

attaching run with latest release:
<img width="1114" alt="Screenshot 2024-04-05 at 22 23 53" src="https://github.com/1arp/create-a-file-action/assets/42278019/afbd0a24-5322-4a9c-b4d4-ec71b671a047">

**Comment by tueda** [comment]
Thank you for the prompt fix! I've confirmed that 0.4.5 works well [(log)](https://github.com/tueda/test-create-a-file-action/actions/runs/8575911281).
